api/src/main/java/com/xwiki/googleapps/GoogleAppsScriptService.java

```diff
@@ -65,7 +63,8 @@
      * @since 3.0
      */
     @Unstable
-    public boolean isActive() throws XWikiException {
+    public boolean isActive() throws XWikiException
```

Comment:
Script services usually don't throw exceptions because it's a pain to use ``#try`` everywhere.